modules/dom-manipulator.js

```diff
@@ -0,0 +1,69 @@
+import Time from './time.js';
+
+class DomManipulator {
+  constructor(removeBook) {
+    this.removeBook = removeBook;
+    this.bookContainer = document.querySelector('.book-container');
+    this.bookContainer.innerHTML = '';
+    this.timeElement = document.getElementById('time');
+    this.navLinks = document.querySelectorAll('.nav-link');
```

Comment:
- [ ] Hi you really did an amazing job but as part of the requirements, you are supposed to use ES6 function components instead of classes. Please kindly change all your classes to ES6 arrow functions